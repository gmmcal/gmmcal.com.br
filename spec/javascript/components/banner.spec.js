import { mount } from '@vue/test-utils'
import Banner from '@/components/banner'

describe('Banner', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(Banner, { sync: true })
  })

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('has data properties', () => {
    const expected = ['I18n']
    const received = Object.keys(wrapper.vm.$data)
    expect(received).toEqual(expected)
  })

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
