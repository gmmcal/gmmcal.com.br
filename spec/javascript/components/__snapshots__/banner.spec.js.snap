// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<div class="banner">
  <div class="row">
    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 no-padd" style="height: 100%;">
      <div class="banner-part">
        <div class="lazy banner-img"></div>
      </div>
    </div>
  </div>
  <div class="banner-text container">
    <h1>Gustavo Cunha</h1>
    <h2>Web developer, Mobile and Software Engineer Consultant</h2>
  </div>
</div>
`;
